File: C/1012.c
#include <stdio.h>

int main(){

    double valor, valor2, valor3;

    scanf("%lf %lf %lf",&valor,&valor2,&valor3);


    printf("TRIANGULO: %.3lf\n",(valor*valor3)/2);
    printf("CIRCULO: %.3lf\n",(3.14159*(valor3*valor3)));
    printf("TRAPEZIO: %.3lf\n",(valor3*(valor+valor2))/2);
    printf("QUADRADO: %.3lf\n",(valor2*valor2));
    printf("RETANGULO: %.3lf\n",(valor*valor2));

    return 0;
}

File: Python/1012.py
abc = input().split(" ")

a, b, c = abc
pi = 3.14159
a = float(a)
b = float(b)
c = float(c)

triangulo = (a* c)/2
circulo = pi * (c * c)
trapezio = c * (a + b) / 2
quadrado = b * b
retangulo = a * b


print("TRIANGULO: %0.3f"%triangulo)
print("CIRCULO: %0.3f"%circulo)
print("TRAPEZIO: %0.3f"%trapezio)
print("QUADRADO: %0.3f"%quadrado)
print("RETANGULO: %0.3f"%retangulo)
